src/js/components/Page.js

```diff
@@ -105,7 +105,7 @@ var Page = React.createClass({
         {this.getPageHeader(title, navigation)}
         <GeminiScrollbar autoshow={true} className="page-content
           container-scrollable inverse" ref="pageRef">
-          <div className="flex-container-col container container-fluid container-pod container-pod-short-top">
+          <div className="flex-container-col flex-shrink flex-grow container container-fluid container-pod container-pod-short-top">
```

Comment:
This did not fix the log view issues that we experienced and it had other implications on the application: https://github.com/dcos/dcos-ui/pull/228. Let's make sure to fix this properly.